src/routes.ts

```diff
@@ -12,27 +12,27 @@ const routes = {
     isPrivate: true,
   },
   login: {
-    path: "/login" as const,
```

Comment:
Because we are not using `readonly` properties anywhere in the app (interfaces etc), or are we using it here for another reason?

Reply:
The reasoning is not related to readonly but to type inference. The `const assertion` makes the compiler infer the most specific possible type, so in this case, `login.path` would be inferred as `"/login"` instead of `string`. This is nice when you use it later:

<img width="600px" src="https://user-images.githubusercontent.com/15896177/182657268-5019dbca-2dbf-4b1f-8e47-bb98988dbc18.gif">